django_irods/views.py

```diff
@@ -53,7 +53,7 @@ def download(request, path, use_async=True, use_reverse_proxy=True,
     6. a previously zipped file that was zipped asynchronously.
 
     """
-    if __debug__:
```

Comment:
There are tons of other places we are using `__debug__`.  If we remove the if statement, we can update the logging settings to write the debug logs for the package as needed.  We aren't writing DEBUG logs by default in production anyways.